augmentedreality/Common/source/ArcGISArViewInterface.cpp

```diff
@@ -37,8 +37,8 @@ using namespace Esri::ArcGISRuntime::Toolkit::Internal;
   \brief Base class to impement AR scene view.
 
   This class provides the AR features without using the C++ or QML APIs of the ArcGIS Runtime SDK for Qt.
-  It is used as a base class to create two API-dependent classes: \l ArcGISArSceneView which uses the C++ API and
-  \l QmlArcGISArSceneView which uses the QML API.
+  It is used as a base class to create two API-dependent classes: \l [CPP] ArcGISArView which uses the C++ API and
+  \l [QML] ArcGISArView which uses the QML API.
```

Comment:
Must use the name of QML component, not the internal C++ class. As the name is now ambiguous between C++ and QML APIs, use qualifiers to fix the link.